### ISSUE TITLE
docker always needs to run with `--generate_export_script_only`

### ISSUE THREAD

**Issue by marrip** [InPreD/TSOPPI_SADET#16]
If I understand your code correctly, you actually need to use `--generate_export_script_only` if you run the container image as the paths in the generated script would not match the paths used in the container, right?

**Comment by danielvo** [comment]
Since the file paths inside and outside the container may differ (based on how the volume mounting was done by the user), the bash script content that references specific files needs to differ based on whether it is supposed to be executed from within the image or outside of it. The "--generate_export_script_only" option should lead to: 1) creating a bash script that should be working as intended outside of the image, 2) not executing that script from within the image (executing it might not work, depending on the volume mounting). If the option is omitted, the paths within the bash script will refer to file paths as they are seen within the image, and the script will be executed (delayed execution on the host system wouldn't necessarily make sense, as the paths may differ).

**Comment by marrip** [comment]
ah, true, I didn't go back to see that you are using the bool to decide wether or not the script paths are adjusted for or not. Sorry about that. Would it make sense to open the export script up so that paths can be specified when calling it instead of hard-coding them? It could be simply positional arguments indicating the export file list and the password file and anything else required. That would allow us as well to reuse the script once it is generated. Just an idea as it would resolve the whole path handling issue a bit ☺️

**Comment by marrip** [comment]
True, one would have to specify paths several times. However, it would omit a situation where a user creates an export script to run manually and then trying to run the script within the container. I don't think it is a big issue but it is a bit confusing. I guess in general it would be good to check how big the differences are timewise running the export script on the host system, inside the container, via python and actually using python packages to run the packaging/encryption. The outcome would enable us to make the best decision for how to package and encrypt and we can make an informed decision about how we want to handle the setting of paths.
